metis-indexing/metis-indexing-client/src/test/java/eu/europeana/indexing/client/test/PublishingServiceTest.java

```diff
@@ -0,0 +1,144 @@
+package eu.europeana.indexing.client.test;
+
+import eu.europeana.indexing.client.config.IndexingConfig;
+import eu.europeana.indexing.service.PublishingService;
+import eu.europeana.indexing.service.dao.FullBeanDao;
+import eu.europeana.indexing.service.mongo.MorphiaDatastoreProvider;
+import eu.europeana.metis.exception.IndexingException;
+import eu.europeana.metis.mongo.EmbeddedLocalhostMongo;
+
+import java.io.IOException;
+
+import org.apache.solr.client.solrj.impl.CloudSolrServer;
+import org.apache.solr.client.solrj.impl.LBHttpSolrServer;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import com.mongodb.MongoClient;
+
+public class PublishingServiceTest {
```

Comment:
This class doesn't seem to test much at all! 

Also, I am not a fan of testing large string inputs, I prefer to test more specifically constructed values that I can then verify. But I suppose that this could be seen as a matter of personal opinion.

Reply:
I prefer to do meaningful unit tests using mocks instead of superficial ones. In this case, usage of corelib is preventing the writing of meaningful unit tests with mocks. We'll have to decide as a team how we want to approach mocking.